Turkey/CsprojCompatibilityPatcher.cs

```diff
@@ -14,15 +14,8 @@ public string Patch(string originalCsprojContents, Version newRuntime)
         {
             var pattern = @"<TargetFramework>net(?:coreapp)?\d\.\d+</TargetFramework>";
             var versionString = newRuntime.MajorMinor;
-            string replacement = null;
-            if (newRuntime.Major < 4)
-            {
-                replacement = $"<TargetFramework>netcoreapp{versionString}</TargetFramework>";
-            }
-            else
-            {
-                replacement = $"<TargetFramework>net{versionString}</TargetFramework>";
-            }
+            var replacement = $"<TargetFramework>net{versionString}</TargetFramework>";
```

Comment:
Is there a reason to keep `CsprojCompatibilityPatcher`?

I think we no longer need it since https://github.com/redhat-developer/dotnet-bunny/pull/84.

cc @omajid

Reply:
Mistake on my part, I meant to remove it. Thanks!

Reply:
I've removed them.